app/assets/stylesheets/apply.scss

```diff
@@ -67,6 +67,39 @@
   z-index: -1;
 }
 
+.authorize-icon {
+  font-size: 3em;
+  position: absolute;
+}
+
+.fb-authorize-icon {
+  bottom: 156px;
+  left: 39px;
+}
+
+.google-authorize-icon {
+  bottom: 68px;
+  left: 39px;
+}
+.fb-authorize-button {
+  width: 100%;
+  background: #3b5998;
```

Comment:
Color literals like `#3b5998` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#3b5998` should be written as `#3B5998`